analyze/lib/Plugin.ts

```diff
@@ -63,16 +68,11 @@ export default abstract class Plugin {
     for (const probeBucket of probeBuckets) {
       for (const probe of probeBucket.probes) {
         if (probe.checkType !== checkType) continue;
-        const toCheck = checksById[probe.check.id];
+        const toCheck = checksBySignature[probe.check.signature];
         const humanScore = probe.check.generateHumanScore(toCheck, profileCountOverTime);
-        // if (humanScore < 100 && probe.id === 'auto-aaad') {
-        //   console.log(probe.check.id, probe.check);
-        //   console.log('-----------')
-        //   const toChecks = checks.filter(x => x.idPrefix === probe.check.idPrefix);
-        //   const consoleChecks = toChecks.length ? toChecks : checks.map(x => ({ idPrefix: x.idPrefix, ...x }));
-        //   console.log(toCheck || consoleChecks);
-        // }
```

Comment:
These 7 lines of code have been my go-to method for debugging failed probes.

Reply:
Now that we're including the failed check in the results, I thought this wasn't necessary. Do you want to keep it?

Reply:
Hopefully that works. I'll add it back in if needed.